### DISCUSSION TITLE
Statistics UI

### DISCUSSION THREAD

**Discussion by bf-tom** [betfinio/public#48] [Bugs and Reports]
<img width="951" alt="Screenshot 2024-10-03 at 15 13 10" src="https://github.com/user-attachments/assets/fdd22f82-26e8-4f83-8ece-a71fcd7ff28a">

**Comment by JackHipeek** [comment]
Total revenue didnt show the statitic. just a direct LINE. 
<img width="1194" alt="Snímek obrazovky 2024-10-04 v 0 35 55" src="https://github.com/user-attachments/assets/551f7392-ab1a-49a5-9697-d3573f9cc2ef">

**Comment by bf-monster** [comment]
This page is still improving. Please, stay put for updates
